<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1101</width>
      <height>831</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>7</number>
    </property>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="ent_reg">
      <property name="geometry">
       <rect>
        <x>144</x>
        <y>215</y>
        <width>771</width>
        <height>71</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Войдите или зарегистрируйтесь</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="ent_btn">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>330</y>
        <width>341</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Войти</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reg_btn">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>330</y>
        <width>341</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Зарегистрироваться</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="reg_text">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>250</y>
        <width>451</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Регистрация</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="logr_text">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>310</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Введите логин:</string>
      </property>
     </widget>
     <widget class="QLabel" name="pasr_text">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>390</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Введите пароль:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
     <widget class="QPushButton" name="to_reg_btn">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>480</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Зарегестрироваться</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="login_reg">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>330</y>
        <width>491</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="pass_reg">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>410</y>
        <width>491</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="error">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>560</y>
        <width>491</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="ent_text">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>250</y>
        <width>451</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Вход</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="login_ent">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>340</y>
        <width>491</width>
        <height>51</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="log_text">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>310</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Логин</string>
      </property>
     </widget>
     <widget class="QLabel" name="pas_text">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>400</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Пароль</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
     <widget class="QPushButton" name="to_ent_btn">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>490</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Войти</string>
      </property>
     </widget>
     <widget class="QPushButton" name="forget_password">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>550</y>
        <width>331</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Я забыл пароль</string>
      </property>
     </widget>
     <widget class="QLabel" name="message">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>590</y>
        <width>331</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="pass_ent">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>420</y>
        <width>491</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QComboBox" name="topics">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>360</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Выбор двух лишних вариантов (задание 1)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Выбор из списка (задание 2)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Выбор из списка (задание 4)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Выбор из списка (задание 5)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Выбор из списка (задание 7)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Выбор из списка (задание 8)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Выбор из списка (задание 10)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Выбор из списка (задание 11)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Выбор из списка (задание 12)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Выбор характерных черт из списка (задание 14)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Выбор из списка (задание 16)</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="top_text">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>320</y>
        <width>371</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Темы заданий</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="add_task_btn">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>480</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить задание</string>
      </property>
     </widget>
     <widget class="QLabel" name="task_added_txt">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>550</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="profile_btn">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Профиль</string>
      </property>
     </widget>
     <widget class="QPushButton" name="make_var">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>420</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Сгенерировать вариант</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="answer">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>520</y>
        <width>581</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="else_task">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>610</y>
        <width>111</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Пропустить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ans_btn">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>610</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Решить</string>
      </property>
     </widget>
     <widget class="QLabel" name="task">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>951</width>
        <height>431</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="end_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>780</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Завершить</string>
      </property>
     </widget>
     <widget class="QLabel" name="comm">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>700</y>
        <width>581</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="back3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="res_text">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ваш результат:</string>
      </property>
     </widget>
     <widget class="QLabel" name="wa_text">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>351</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Задания, которые Вы решили неправильно:</string>
      </property>
     </widget>
     <widget class="QLabel" name="out_of">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>80</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>из</string>
      </property>
     </widget>
     <widget class="QLabel" name="point_text">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>80</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableWidget" name="wa_texts">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>150</y>
        <width>421</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="ya_num">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>80</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="all_a_num">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>80</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_7">
     <widget class="QLineEdit" name="task_add">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>90</y>
        <width>651</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="answer_add">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>590</y>
        <width>651</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="topics_add">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Тема 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Тема2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Тема 4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Тема 5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Тема 7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Тема 8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Тема 10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Тема 11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Тема 12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Тема 14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Тема 16</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="add">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>650</y>
        <width>181</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить задание</string>
      </property>
     </widget>
     <widget class="QLabel" name="error1">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>650</y>
        <width>341</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="back4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_8">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>1111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Профиль</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="change_pass">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>660</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Изменить пароль</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>260</y>
        <width>1111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Количество правильно решенных задач:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="num_tasks">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>140</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableWidget" name="stat_table">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>330</y>
        <width>401</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="progress">
      <property name="geometry">
       <rect>
        <x>354</x>
        <y>150</y>
        <width>411</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_9">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>210</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Введите логин:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>280</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Введите новый пароль:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="login">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>240</y>
        <width>501</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="new_pass">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>310</y>
        <width>501</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="error_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>390</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="update_pass">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>350</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Обновить пароль</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1121</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
